### PR TITLE
Resume active subscription doesn't cause error anymore.

### PR DESCRIPTION
**Pull request by ekoval**
Resume active subscription doesn't cause error anymore.
Due to recent API changes.

### CODE REVIEW

File: tests/campaigns/test_integration.py [hunk 1]
```diff
@@ -208,14 +208,14 @@ def setUp(self):
...
+        self.result1 = service.resume_campaign_subscription(
             self.account_token, self.campaign_id)
 
-    def test_error_400_is_raised(self):
-        # And try to resume it again (Active->Active).
-        self.assertHTTPErrorIsRaised(
-            SitewitService().resume_campaign_subscription, (
-                self.account_token, self.campaign_id), 400)
+        self.result2 = SitewitService().resume_campaign_subscription(
+                self.account_token, self.campaign_id)
```
Comment (snitch) [inline tests/campaigns/test_integration.py]: E126 continuation line over-indented for hanging indent

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
:+1:

**Pull request merged**